### DISCUSSION TITLE
Custom Panel

### DISCUSSION THREAD

**Discussion by rekabuk** [foxglove/community#786] [General]
Hi,
I have created a custom panel (on my dev machine) that can be loaded by the "FoxGlove APP" running on my dev machine, which is connecting via Rosbridge to my ROS2 robot also running on my dev machine.
But how can I make that panel accsesible to a different machine (Tablet) that connects through https://studio.fogglove.dev and Rosbridge to ROS2 on my dev machine?
I hope that made sense?
Regards
Andrew

**Comment by defunctzombie** [comment]
You have two options which are documented here: https://foxglove.dev/docs/studio/extensions/publish#sharing-with-your-team

You can either use your Foxglove account to publish the panel to your account and it will be available when you sign-in to your account on either machine. Or you can publish it to the public marketplace.

**Comment by rekabuk** [comment]
Thank-you, I'm sure I read a similar looking page yesterday which said this process was not complete? Must have been an old page.
Andrew
